src/components/test/Checkbox-test.js

```diff
@@ -65,7 +65,7 @@ describe('LabeledCheckbox', () => {
   it('renders a checkbox SvgIcon for styling the checkbox visually', () => {
     const wrapper = createComponent();
     const icon = wrapper.find('SvgIcon');
-    assert.equal(icon.props().name, 'hyp-checkbox');
+    assert.equal(icon.props().name.toString(), Symbol('checkbox').toString());
```

Comment:
This kosher?

Reply:
Yes, I think this is fine, as long as we only care about the symbol's description here rather than symbol equality.